tests/v2/validation/rbac/rbac_test.go

```diff
@@ -335,6 +335,68 @@ func (rb *RBTestSuite) ValidateAddCMAsProjectOwner() {
 
 }
 
+func (rb *RBTestSuite) ValidateAddPOsAsProjectOwner() {
+	//Create project as a cluster Owner
```

Comment:
nit; can we remove these comments? The function names are descriptive enough so no need for the additional comments.